README.md

```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
 mysql> create database userdb;
 ```
 
+**Step 3: Installing MySQL drivers for node**
+
+We want to use MySQL with JavaScript and to this end, we use `mysqljs` driver.
```

Comment:
I'd add a hyperlink to the documentation: https://github.com/mysqljs/mysql.

Reply:
Added in the new commit.